services/DrugService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+
+    DrugService
+
+*/
+
+namespace services;
```

Comment:
change to:
namespace OpenEMR\Services;

Reply:
disregard above comment for now. I am in process of converting this namespace, so can do this in the future.